merge/entrypoint.py

```diff
@@ -178,6 +178,17 @@ def main():
     print(f"Adding {pr_author} to collaborators")
     repo.add_to_collaborators(pr_author, permission="push")
 
+    print("Add trusted maintainers to collaborators")
+    trusted_maintainers = flathub.get_team_by_slug("trusted-maintainers")
+    trusted_maintainers.add_to_repos(repo)
+    trusted_maintainers.update_team_repository(repo, "push")
```

Comment:
Do we need the second one, i.e. add the repo to the team repos?

Reply:
Or vice versa: from the [documentation](https://docs.github.com/en/rest/reference/teams#add-or-update-team-membership-for-a-user-legacy), unless I've confused something, it seems like the `add_to_repos` endpoint is a legacy one, to be replaced by the `update_team_repository` one.

Reply:
Hm, I'm not getting a deprecation notice from pygithub itself, but you are right that `update_team_repository` is enough. Good catch, I will fix it.